lib/rdf2marc/model2marc/field_647.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Rdf2marc
+  module Model2marc
+    # Maps model to field 647.
+    class Field647 < Field
+      def initialize(marc_record, model)
+        super(marc_record, model, '647')
+      end
+
+      def build
+        field.indicator2 = Thesaurus.code_for(model.thesaurus)
+        append('a', model.name)
+        append('2', model.source)
+        append_repeatable('0', model.uris)
+      end
```

Comment:
I don't know when we go for complete lists of possible subfields from the marc spec https://www.loc.gov/marc/bibliographic/bd647.html -- see 650, 651, etc. -- and when we don't.

Reply:
Yeah, I don't remember the strategy so I'd say this is OK. At the very least if we map from RDF to model, we should map model to MARC.